src/DonationForms/V2/DonationFormsAdminPage.php

```diff
@@ -202,4 +218,34 @@ public static function getUrl()
     {
         return add_query_arg(['page' => 'give-forms'], admin_url('edit.php?post_type=give_forms'));
     }
+
+    /**
+     * Get an array of unsupported addons for addons compatibility modal
+     *
+     * @unreleased
+     * @return array
+     */
+    public function getUnsupportedAddons(): array
+    {
+        $unsupportedAddons = [
```

Comment:
@alaca We need to fill out this unsupported list before we merge.

Reply:
@kjohnson I'm not sure which ones are supported or not, so I will add them all for now.

Reply:
List updated https://github.com/impress-org/givewp/pull/6832/commits/c3e912ca9f61d7481de447d067dc8cbd46ac0228